ui/uihandler.go

```diff
@@ -845,7 +845,13 @@ func Serve(baseurl, port, debugpath, uipath, staticRsrcDir string, datadir strin
 	}
 	debugPath = ".." + debugpath // TODO: calculate actual path if not same number of directories
 	http.HandleFunc(uiPath, Handler)
-	fmt.Printf("UI starting - visit:\nhttp://%s%s\n", urlHostPort, uiPath)
+	switch {
+	case strings.Contains(port, ":"):
+		fmt.Printf("UI starting - visit:\nhttp://%s%s\n", urlHostPort, uiPath)
+	default:
+		fmt.Printf("UI starting - visit:\nhttp://%s%s"+
```

Comment:
let's keep the common part together, remove the \n and then just print "   (or any host/ip reaching this machine)\n" or "\n"